Dockerfile

```diff
@@ -177,11 +177,14 @@ ENV _APP_SERVER=swoole \
     _APP_SMTP_SECURE= \
     _APP_SMTP_USERNAME= \
     _APP_SMTP_PASSWORD= \
+    _APP_FUNCTIONS_SIZE_LIMIT=30000000 \
     _APP_FUNCTIONS_TIMEOUT=900 \
     _APP_FUNCTIONS_CONTAINERS=10 \
     _APP_FUNCTIONS_CPUS=1 \
     _APP_FUNCTIONS_MEMORY=128 \
     _APP_FUNCTIONS_MEMORY_SWAP=128 \
+    _APP_EXECUTOR_SECRET=a-random-secret \
+    _APP_EXECUTOR_RUNTIME_NETWORK=appwrite_runtimes \
```

Comment:
In the `docker-compose.yml` file we already have a network called `runtimes`, what is the difference between them? Can we user a term that will help differentiate between the two?

Reply:
They are the same network, the `appwrite_` prefix is because it's apart of the appwrite docker project